### PR TITLE
types/mydecimal_test: add tests for round with different round_mode

### PR DESCRIPTION
**Pull request by AndreMouche**
Hi,
     This PR adds tests for round with different `round_mode`.

@XuHuaiyu @zz-jason PTAL

### CODE REVIEW

**Comment by XuHuaiyu** [comment]
LGTM

**Comment by XuHuaiyu** [comment]
PTAL @zz-jason

File: util/types/mydecimal_test.go [hunk 1]
```diff
@@ -297,6 +297,83 @@ func (s *testMyDecimalSuite) TestRound(c *C) {
 	doTest(c, tests)
 }
 
+func (s *testMyDecimalSuite) TestRoundWithTruncate(c *C) {
+	type tcase struct {
```
Comment (ngaut) [inline util/types/mydecimal_test.go]: This is not the best practice of writing tests. 
DRY.

Comment (ngaut) [inline util/types/mydecimal_test.go]: Please see https://golang.org/doc/code.html#Testing

Comment (AndreMouche) [inline util/types/mydecimal_test.go]: It seems the style of this file, I'm not pretty sure whether we should keep consistent with the above tests. Maybe we could update the style in another independent PR == @ngaut

**Comment by AndreMouche** [comment]
@XuHuaiyu @zz-jason @ngaut  PTAL again

**Pull request merged**